pkg/providers/cohere/chat.go

```diff
@@ -107,46 +107,49 @@ func (c *Client) doChatRequest(ctx context.Context, payload *ChatRequest) (*sche
 	// Build request payload
 	rawPayload, err := json.Marshal(payload)
 	if err != nil {
-		return nil, fmt.Errorf("unable to marshal cohere chat request payload: %w", err)
```

Comment:
@mkrueger12 this is cohere package and the change in this package replace "cohere" by "openai". 
Is this intentionally?

Reply:
fixed